datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -108,6 +108,8 @@ pub enum LogicalPlan {
     Distinct(Distinct),
     /// Set a Variable
     SetVariable(SetVariable),
+    /// Prepare a statement
+    Prepare(Prepare),
```

Comment:
Is the plan to add a `Execute` statement as a follow on PR?

Reply:
I think so. It will be cleaner unless you say otherwise

Reply:
Follow on PR sounds good to me 👍